pom.xml

```diff
@@ -16,7 +16,7 @@
   <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.version>2.332.1</jenkins.version>
+    <jenkins.version>2.346.1</jenkins.version>
```

Comment:
First version of core that contains https://github.com/jenkinsci/lib-version-number/pull/41, which I am using in this code.